packages/coding-standards/src/Phpstan/OrmPropertyGetterAndSetterHasNoTypehintRule.php

```diff
@@ -70,6 +89,49 @@ public function processNode(Node $node, Scope $scope): array
         return [];
     }
 
+    /**
+     * @param \PhpParser\Node\Stmt\ClassMethod $node
+     * @param string $methodName
+     * @param \PHPStan\Analyser\Scope $scope
+     * @param \PHPStan\Reflection\PropertyReflection $propertyReflection
+     * @return array
+     */
+    private function checkSetter(
+        ClassMethod $node,
+        string $methodName,
+        Scope $scope,
+        PropertyReflection $propertyReflection,
+    ): array {
+        if (count($node->getParams()) !== 1) {
+            return [];
+        }
```

Comment:
Why do you want to skip it for setters with two and more parameters?

Reply:
Because then it's not just a simple setter and I don't want to make any assumptions about such code